examples/notebooks/langchain_evaluation.ipynb

```diff
@@ -0,0 +1,715 @@
+       "   'reference': {'answer': 'Montresor is insulted by Lenora and seeks revenge.'}}}}"
+      ]
+     },
+     "execution_count": 60,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from langsmith import Client\n",
+    "from langchain.evaluation import EvaluatorType\n",
+    "from langchain.smith import RunEvalConfig, run_on_dataset\n",
+    "\n",
+    "evaluation_config = RunEvalConfig(\n",
+    "    # LangChain offers several QA Evaluator types\n",
+    "    evaluators=[\n",
+    "        \"qa\",\n",
+    "        \"context_qa\",\n",
+    "        \"cot_qa\",\n",
+    "    ],\n",
+    "    prediction_key=\"result\",\n",
+    ")\n",
+    "\n",
+    "client = Client()\n",
+    "run_on_dataset(\n",
+    "    dataset_name=dataset_name,\n",
+    "    llm_or_chain_factory=create_qa_chain(llm=model, vstore=vstore),\n",
+    "    client=client,\n",
+    "    evaluation=evaluation_config,\n",
+    "    verbose=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Open the link to view your evaluation results. Note that the first two queries should have \"incorrect\" results, as the dataset purposely contained incorrect answers for those. \n",
+    "\n",
+    "For more details, see [LangSmith Testing and Evaluation](https://docs.smith.langchain.com/category/testing--evaluation)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Cleanup"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# WARNING: This will delete the collection and all documents in the collection\n",
+    "# vstore.delete_collection()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nbmake": {
+     "post_cell_execute": [
```

Comment:
not sure if a "post_cell_execute" will run on a "markdown" cell

Reply:
Good catch. Tested this and you are correct.